projects/suricata/project.yaml

```diff
@@ -1,5 +1,5 @@
 homepage: "https://suricata-ids.org"
-language: c++
+language: rust
```

Comment:
so that we get rust llvm-cov

Reply:
I'm not sure how this works.
Here's my understanding.
suricata's c++ code is compiled with clang 14. suricata's rust code is compiled with clang 13.
Are the profraw files produced by suricata coverage builds compatible with clang 13 but not clang 14?